app/controllers/studio_assessments_controller.rb

```diff
@@ -5,6 +5,24 @@ class StudioAssessmentsController < ApplicationController
   # GET /studio_assessments.json
   def index
     @studio_assessments = StudioAssessment.all
+    @studio_list = []
+    @studio_assessments.each do |s|
+      curr = {
+        "name" => s.participant.full_name,
+        "bigpicture_score" => s.bigpicture_score,
+        "progfundamentals_score" => s.progfundamentals_score,
+        "versioncontrol_score" => s.versioncontrol_score,
+        "react_score" => s.react_score,
+        "node_score" => s.node_score,
+        "db_score" =>s.db_score,
+        "problemsolving_score" => s.problemsolving_score,
+        "problemsolvingalt_score" => s.problemsolvingalt_score,
+        "id" => s.participant_id,
```

Comment:
is it necessary to create new objects? can we just use StudioAssessment.all ?

Reply:
nah i was just playing around with some stuffs

Reply:
wait i remeber why I made it like that. To get the name I used participant.full_name, which is a function that gets the full name.